frontend/app/src/components/accounts/AccountBalanceTable.vue

```diff
@@ -596,6 +596,21 @@ const liabilities = (address: string) => {
   return get(accountLiabilities(address));
 };
 const loopringBalances = (address: string) => get(getLoopringBalances(address));
+
+const removeCollapsed = ({ derivationPath, xpub }: XpubPayload) => {
+  const index = get(collapsedXpubs).findIndex(
+    collapsed =>
+      collapsed.derivationPath === derivationPath && collapsed.xpub === xpub
+  );
+
+  if (index >= 0) {
+    get(collapsedXpubs).splice(index, 1);
+  }
+};
+
+defineExpose({
+  removeCollapsed
+});
```

Comment:
Is this something that we broke when moving to composition? how did it work before?

Reply:
Ok so we didn't expose it and we accidentally removed the function right?